test/dataset_collection/test_dataset_collection.py

```diff
@@ -377,3 +361,69 @@ def test_start_end_inline_without_args(self):
                                                                           self.temp_filename)
         self.assertEqual([30, 34], pred_inline_rel_bounds,
                          msg='Wrong inline bounds: {}'.format(pred_inline_rel_bounds))
+
+    def test_inline_with_return_without_assigning(self):
+        filepath = self.current_directory / 'InlineExamples' / 'Parameters.java'
+        test_filepath = self.current_directory / 'InlineTestExamples' / 'Parameters_without_return.java'
+        algorithm_type = InlineTypesAlgorithms.WITH_RETURN_WITHOUT_ARGUMENTS
+        algorithm_for_inlining = AlgorithmFactory().create_obj(algorithm_type)
+        ast = AST.build_from_javalang(build_ast(filepath))
+        m_decl = [
+            x for x in ast.get_proxy_nodes(ASTNodeType.METHOD_DECLARATION)
+            if x.name == 'btSelectMethod'][0]
+        body_start_line, body_end_line = method_body_lines(m_decl, filepath)
```

Comment:
Here are some code duplication among your test function. Might be better to extract.

Reply:
unfortunately, we have side effects here. Also, there are at least 3 variables are used after invocation.
 Seems it's a bad idea. Moreover, it's a test but not functionality of some framework.
 Finally, if I extract that functionality, it doesn't follow single responsibility rule

Reply:
I tried to extract some code. But I do not know whether it's a good idea